test/parallel/test-inspect-support-for-node_options.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const common = require('../common');
+const cluster = require('cluster');
+const assert = require('assert');
+
+common.skipIfInspectorDisabled();
+
+checkForInspectSupport('--inspect');
+
+function checkForInspectSupport(flag) {
+
+  const nodeOptions = JSON.stringify(flag);
+  process.env.NODE_OPTIONS = flag;
+
+  if (cluster.isMaster) {
+    for (let i = 0; i < 2; i++) {
+      cluster.fork();
+    }
+
+    cluster.on('online', (worker) => {
+      worker.disconnect();
+    });
+
+    cluster.on('exit', (worker, code, signal) => {
+      if (worker.exitedAfterDisconnect === false) {
```

Comment:
Should this be an assert rather than if-fail?

Reply:
Does this look ok?
```js
    cluster.on('exit', common.mustCall((worker, code, signal) => {
      assert(worker.exitedAfterDisconnect === true, `For NODE_OPTIONS failed to start cluster`);
    }, 2));
```